src/columns.jl

```diff
@@ -210,6 +219,8 @@ getindex(c::Columns{D}, i::Integer) where {D<:NamedTuple} = D(ith_all(i, c.colum
 
 getindex(c::Columns, p::AbstractVector) = Columns(map(c->c[p], c.columns))
 
+view(c::Columns, I) = Columns(map(a->view(a, I), c.columns))
+
```

Comment:
Or:
```
function view(c::Columns{D,C}, I) where {D,C} 
     cols = map(a->view(a, I), c.columns)
     Columns{D,typeof(cols)}(cols)
 end
```
Not sure what are the implication of the explicit typing.

Reply:
They should be equivalent.

As long as `view` on the columns are inferrable, this will work.